lib/crawly/parsers/links_extractor.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Crawly.Parsers.RequestsExtractor do
+  @moduledoc """
+  Links extractor parser helper, which simplifies the process
+  of links extraction.
+  """
+
+  alias Crawly.Spider.Parse
+
+  def parse(parse_struct) do
+    %Parse{response: response, selector: selector} = parse_struct
```

Comment:
was removed in main PR.

This module should  implement the `Crawly.Pipeline` behaviour, which will then have a `run/3` function that is then piped. The 2nd arg, state, will have a `:response` key and a `spider_name` key. Opts will contain the `:selector`